calpool-be/app.py

```diff
@@ -34,6 +35,37 @@ def nothing():
 def pingpong():
     return "pong"
 
+@app.route('/create_trip', methods=['POST'])
+def create_carpool():
+    data = request.json
+    start_location = data['pickup']
+    end_location = data['destination']
+    start_time = data['depart']
+    end_time = data['arrive']
+    lower_bound = data['lowerBound']
+    upper_bound = data['upperBound']
+    max_people = data['people']
+    comments = data['comments']
+
+    # Hardcoded
+    current_user = User.objects(email="fake@gmail.com").first()
+    # current_user = User.objects(id=get_id()).first()
+
+    new_trip = Trip(
+        start_location=start_location,
+        end_location=end_location,
+        start_time = datetime.strptime(start_time, '%m/%d/%y %H:%M'),
+        end_time= datetime.strptime(end_time, '%m/%d/%y %H:%M'),
+        lower_bound=lower_bound,
+        upper_bound=upper_bound,
+        max_people=max_people,
+        comments=comments,
+        owner=current_user)
+
+    new_trip.save()
+
+    return "done"
```

Comment:
I think it'd be useful to return the Trip id to be a little more descriptive instead. you can follow what lucas did for user id